### ISSUE TITLE
Compatible with existing texture packs?

### ISSUE THREAD

**Issue by xplshn** [Roadhog360/Et-Futurum-Requiem#354]
This is just a question.

**Comment by Roadhog360** [comment]
This is not an issue, you can join the Discord or use the comment section of CurseForge for questions. Please stop submitting non-issues to the *issue* tracker.

That being said, it should be, it uses the vanilla Minecraft domain and is designed with partial compatibility in mind. You may have to rename some folders in the texture pack for it to work. (eg rename "items" to "item" and "blocks" to "block")

Basically ALL of the new added content by Et Futurum requiem uses the modern naming format, however textures for blocks in 1.7.10 might need to be renamed, as many of them use different names from their modern counterparts. Beds are one of the exceptions, they do not use their modern TileEntity texture format at all and still are a standard block renderer so they use block textures.

TL;DR Maybe, you will probably have to rename some things, please stop posting issues that aren't issues on my issue tracker, thanks.
